client/my-sites/site-settings/form-jetpack-monitor.jsx

```diff
@@ -1,212 +1,199 @@
-						id="jetpack_monitor_wp_note"
-						checkedLink={ this.linkState( 'wpNoteNotifications' ) }
-						name="jetpack_monitor_wp_note" />
-					<span>
-						{ this.translate( 'Send notifications via WordPress.com notification.' ) }
-					</span>
-				</FormLabel>
+			this.props.activatingMonitor ||
+			this.props.deactivatingMonitor ||
+			this.props.updatingMonitorSettings ||
+			this.props.fetchingJetpackModules ||
+			this.props.requestingMonitorSettings
 		);
-	},
-
-	// updates the state when an error occurs
-	handleError: function() {
-		this.setState( { submittingForm: false, togglingModule: false } );
-		this.props.markSaved();
-	},
+	}
 
-	disableForm: function() {
-		return this.state.fetchingSettings || this.state.submittingForm || this.props.site.fetchingModules || this.state.togglingModule;
-	},
+	handleActivationButtonClick = () => {
+		const { siteId } = this.props;
+		this.props.activateModule( siteId, 'monitor', true );
+	}
 
-	activateFormButtons: function() {
-		return(
-			<Button
-				compact
-				primary
-				onClick={ this.toggleJetpackModule.bind( this, 'monitor' ) }
-				disabled={ this.disableForm() }
-				>
-				{ this.state.togglingModule ? this.translate( 'Activating…' ) : this.translate( 'Activate' ) }
-			</Button>
-		);
-	},
+	handleDeactivationButtonClick = () => {
+		const { siteId } = this.props;
+		this.props.deactivateModule( siteId, 'monitor', true );
+	}
 
-	deactivateFormButtons: function() {
-		return(
-			<div>
-				<Button
-					compact
-					className="jetpack-monitor__deactivate"
-					onClick={ this.toggleJetpackModule.bind( this, 'monitor' ) }
-					>
-					{ this.state.togglingModule ? this.translate( 'Deactivating…' ) : this.translate( 'Deactivate' ) }
-				</Button>
-				<Button
-					compact
-					primary
-					onClick={ this.saveSettings }
-					>
-					{ this.state.submittingForm ? this.translate( 'Saving…' ) : this.translate( 'Save Settings' ) }
-				</Button>
-			</div>
-		);
-	},
+	render() {
+		const { siteId, translate } = this.props;
 
-	render: function() {
 		if ( ! config.isEnabled( 'settings/security/monitor' ) ) {
 			return null;
 		}
 
 		return (
-			<div>
-				<SectionHeader label={ this.translate( 'Jetpack Monitor' ) }>
-					{ this.state.enabled
-						? this.deactivateFormButtons()
-						: this.activateFormButtons()
+			<div className="site-settings__security-settings">
+				<QueryJetpackModules siteId={ siteId } />
+				<QuerySiteMonitorSettings siteId={ siteId } />
+
+				<SectionHeader label={ translate( 'Jetpack Monitor' ) } />
 
-					}
-				</SectionHeader>
 				<Card className="jetpack-monitor-settings">
-					{
-						this.state.enabled
-						? this.settings()
-						: this.prompt()
-					}
+					<div className="site-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink href={ 'https://jetpack.com/support/monitor/' } icon target="_blank">
+								{ translate( 'Learn more about Monitor.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Learn more about CSS' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).